addons/todo/static/src/views/todo_form/todo_form_controller.js

```diff
@@ -1,14 +1,55 @@
 /** @odoo-module **/
 
+import { _lt } from "@web/core/l10n/translation";
 import { FormController } from "@web/views/form/form_controller";
 import { TodoEditableBreadcrumbName } from "@todo/components/todo_editable_breadcrumb_name/todo_editable_breadcrumb_name";
+const { onWillStart } = owl;
 
 /**
  *  The FormController is overridden to be able to manage the edition of the name of a to-do directly
  *  in the breadcrumb.
  */
 
-export class TodoFormController extends FormController {}
+export class TodoFormController extends FormController {
+    setup() {
+        super.setup();
+        onWillStart(async () => {
+            this.conversionViewId = await this.model.orm.call(
+                'project.task',
+                'get_conversion_view_id',
+                [],
+            );
+        });
+    }
+
+    get actionMenuItems() {
+        this.archiveEnabled = true;
```

Comment:
Is it really needed? 🤔

Reply:
What do you mean? The line of code or the functionality? I was asked to keep the ability to archive todo, so I used that line to be able to do so.

Reply:
The line of code, I am wondering if we don't have to add an attribute in the view instead of doing that. 🤔

Reply:
Oh indeed! I've removed that line.